themes/finna2/templates/RecordDriver/SolrEad/core.phtml

```diff
@@ -6,11 +6,13 @@
     $audioUrls = [];
     $otherUrls = [];
     foreach ($urls as $url){
+      $type = $url['type'] ?? null;
       if (!empty($url['videoSources']) || !empty($url['embed']) && $url['embed'] == 'video' || $this->recordLink()->getEmbeddedVideo($url['url']) == 'data-embed-iframe'){
         array_push($videoUrls, $url);
-      } elseif (!empty($url['embed']) && $url['embed'] == 'audio') {
```

Comment:
Pelottava muutos, ei taaksepäin yhteensopiva. Jos toimintalogiikkaa on tarpeen muuttaa, pitäisi siitä tehdä erillinen PR, joka ei liity R2:een.

Reply:
Jeps, en tajunnut että tätä käytetään muissa ajureissa. Palautin embed-parametrin käytön tähän. Asettaminen täällä:
https://github.com/NatLibFi/NDL-VuFind2/pull/1705/files#diff-e7a1054c570f3ace627c53a7d6aa2ec3R958